backend/CircleForms/Models/Posts/Post.cs

```diff
@@ -58,4 +49,15 @@ public Post(string id)
     [JsonProperty("publish_time")]
     [SwaggerSchema(ReadOnly = true)]
     public DateTime PublishTime { get; set; }
+
+    public string GenerateNewID()
+    {
+        return ObjectId.GenerateNewId().ToString();
+    }
+
+    [ObjectId]
+    [BsonId]
+    [JsonProperty("id")]
+    [SwaggerSchema(ReadOnly = true)]
+    public string ID { get; set; }
```

Comment:
Does it have to be full uppercase? Pretty sure that's against the naming convention

Reply:
yep

Reply:
You should add it to the naming exceptions then so that we wont get squiggly lines all the time

Reply:
fix it in your ide of choice